### PR TITLE
Update SampleCreateCircleAndWriteTo3dmFile.py

### PR DESCRIPTION
**Pull request by uozcan12**
corrected Rhino version

### CODE REVIEW

**Comment by uozcan12** [comment]
corrected current Rhino version

File: rhino3dm/py/SampleCreateCircleAndWriteTo3dmFile.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import random
+
+model = rhino3dm.File3dm()
+for i in range(20):
+	pt = rhino3dm.Point3d(random.uniform(-10,10), random.uniform(-10,10), 0)
+	model.Objects.AddPoint(pt)
+	circle = rhino3dm.Circle(pt, random.uniform(1,4))
+	model.Objects.AddCircle(circle)
+	
+model.Write("circle.3dm", 7)
```
Comment (sbaer) [inline rhino3dm/py/SampleCreateCircleAndWriteTo3dmFile.py:11]: sorry, I was unclear. The 7 should be removed so we don't deal with version numbers in the future.
`model.Write("circle.3dm")` should work

Comment (uozcan12) [reply rhino3dm/py/SampleCreateCircleAndWriteTo3dmFile.py:11]: Ok, You are right. I will correct.

**Comment by sbaer** [comment]
Are you making a new PR? I don't understand why this one was closed

**Pull request closed without merging**